.github/workflows/docs.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Build HTML documentation
         run: make docs-html
 
-      - uses: errata-ai/vale-action@reviewdog
+      - uses: errata-ai/-action@reviewdog
```

Comment:
Revert.

Reply:
sir, i did not changed this manually, but if you want me to revert then i will

Reply:
Why did you change it, commit it, and push it to this PR?